ConfigurationSystem/scripts/dirac-admin-add-resources.py

```diff
@@ -73,23 +73,29 @@ def checkUnusedCEs():
 
   gLogger.notice('looking for new computing resources in the BDII database...')
 
-  res = getSiteCEMapping()
+  res = getQueues(community=vo)
   if not res['OK']:
     gLogger.error('ERROR: failed to get CEs from CS', res['Message'])
     DIRACExit(-1)
+
   knownCEs = []
-  for site in res['Value']:
-    knownCEs = knownCEs + res['Value'][site]
+  for _site, ces in res['Value'].items():
+    knownCEs.extend(ces.keys())
```

Comment:
```suggestion
    knownCEs.extend(list(ces))
```

Reply:
Same as above, casting isn't necessary at all, but at least this isn't  a no-op :)
Switching to a `set` as well.